webnode/src/components/consent-overlay/button-group/index.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from "react";
+import color from "colors";
+import Radium from 'radium';
+
+interface ButtonProps {
+  onClick: () => void;
+  children: any;
```

Comment:
for "children" (a react thing) i think any is okay,

but for our own defined types i think we could have "models", so it would look like:

```
import Dog from "../models/dog"
import Cat from "../models/cat"

interface CoolProps {
  dogs: Array<Dog>,
  cats: Object<Cat>,
}
```

Reply:
agree, 

its possible to dont define if its empty <>